src/format.rs

```diff
@@ -33,7 +33,7 @@ impl Default for Format {
     /// green for 200s, yellow for 300s, red for 400s, and bright red for 500s.
     fn default() -> Format {
         fn status_color(_req: &Request, res: &Response) -> Option<color::Color> {
-            match *res.status.as_ref().unwrap_or(&NotFound) as u16 / 100 {
+            match (*res.status.as_ref().unwrap_or(&NotFound)).to_u16() / 100 {
```

Comment:
We should use the `class` method from hyper: http://hyperium.github.io/hyper/hyper/status/enum.StatusCode.html#method.class